proposals/exp-show-target.md

```diff
@@ -0,0 +1,214 @@
+│ └── exp-44136  │ Feb 17, 2021 │  0.5674 │ 0.2           │ 20170428     │ 1500                   │ 2                │ 20170428   │ 50                 │ -                  │ - │
+│ f8e9d93        │ Nov 15, 2020 │ 0.54175 │ 0.2           │ 20170428     │ 1500                   │ 2                │ 20170428   │ 50                 │ -                  │ - │
+│ 377c988        │ Nov 15, 2020 │ 0.54175 │ 0.2           │ 20170428     │ 500                    │ 1                │ 20170428   │ 50                 │ -                  │ - │
+│ 27d4e7c        │ Nov 15, 2020 │       - │ 0.2           │ 20170428     │ 500                    │ 1                │ 20170428   │ 50                 │ -                  │ - │
+│ 8bf2091        │ Nov 15, 2020 │       - │ 0.2           │ 20170428     │ 500                    │ 1                │ 20170428   │ 50                 │ -                  │ - │
+│ ff9e2fa        │ Nov 15, 2020 │       - │ 0.2           │ 20170428     │ 500                    │ 1                │ 20170428   │ 50                 │ -                  │ - │
+│ 11e1705        │ Nov 15, 2020 │       - │ 0.2           │ 20170428     │ 1500                   │ 2                │ 20170428   │ 10                 │ -                  │ - │
+│ a82585e        │ Nov 15, 2020 │       - │ 0.2           │ 20170428     │ 1500                   │ 2                │ 20170428   │ 10                 │ -                  │ - │
+│ c778a78        │ Nov 15, 2020 │       - │ 0.2           │ 20170428     │ 1500                   │ 2                │ 20170428   │ 10                 │ -                  │ - │
+│ 551082e        │ Nov 15, 2020 │       - │ 0.2           │ 20170428     │ 1500                   │ 2                │ 20170428   │ 10                 │ -                  │ - │
+│ 15bef96        │ Nov 15, 2020 │       - │ 0.2           │ 20170428     │ 1500                   │ 2                │ 20170428   │ 10                 │ -                  │ - │
+└────────────────┴──────────────┴─────────┴───────────────┴──────────────┴────────────────────────┴──────────────────┴────────────┴────────────────────┴────────────────────┴───┘
+```
+
+This table includes parameters and metrics from the `evaluate` stage and all
+stages upstream of it.  It's possible to also have a `--single-item` argument to
+exclude upstream changes, but this is out of scope for now.
+
+# How We Teach This
+
+See [Detailed design](#detailed-design) for a proposed help message.
+
+This is consistent with other commands, and it's only one additional argument to
+an existing command, so it likely doesn't require much teaching. It might be
+necessary to explicitly add that parameters and metrics from upstream stages
+will be shown.
+
+# Drawbacks
+
+This proposal may require changes to how DVC collects and tracks parameters and
+metrics. Implementation could break existing functionality, and it is
+inconsistent with the current behaviors of `dvc params`, `dvc metrics`, and `dvc
+exp diff` commands.
```

Comment:
This may be an advantage. Otherwise why have all these different options if they do the same thing?

Reply:
> This may be an advantage.

I agree though we need to remember that we released a major version recently, and this would be kind of a breaking change. We don't know how many users actually want to see current behavior, and if changing it won't make users lose trust in our development process.

Reply:
Ok I guess its addressed down below.